include/openssl/core_names.h

```diff
@@ -27,6 +27,7 @@ extern "C" {
 #define OSSL_PROV_PARAM_STATUS             "status"              /* uint */
 #define OSSL_PROV_PARAM_SECURITY_CHECKS    "security-checks"     /* uint */
 #define OSSL_PROV_PARAM_TLS1_PRF_EMS_CHECK "tls1-prf-ems-check"  /* uint */
+#define OSSL_PROV_PARAM_DRBG_TRUNC_DIGEST  "drbg-no-trunc-md"    /* uint */
```

Comment:
is no supposed to be part of this name? I didnt do this for the ems_check..

Reply:
I think "no" is good here.  Setting the option to "1" means truncated MD are **not** allowed.

The EMS one forces EMS when set to "1".